src/components/HomePage.tsx

```diff
@@ -47,7 +47,7 @@ function SessionInfo({ session: { serverInfo } }) {
   return (
     <div>
       <div className="card server-info-panel">
-        <div className="card-header">
+        <div className="card-header" data-testid="home-header">
```

Comment:
I remember the Testing Library [docs](https://testing-library.com/docs/queries/about/#priority) that they recommend:
> 1. Queries Accessible to Everyone
> 2. Semantic Queries
> 3. Test IDs

Some of the places where we've used test IDs make sense, but I wonder if we could get away with just querying text sometimes 🤔. For example, in the test that uses this test id, could we look for `Server information` text instead?

Reply:
Good call, and we've done that in many other places along with getting inputs based on label text. But clearly  I was on autopilot when I did this one. Adjusting now.

Reply:
Nice. That still leaves a few test ids in this PR -- were those cases where a test id is the most pragmatic query to use?

Reply:
Yes, my first attempt was to retrieve an element based on text. Usually that was in a label or the text of a button, but sometimes it was from static text that was a part of that field.  Where that wasn't possible because content was too dynamic, non-existant (like the rjsf form wrapper), or the could be in multiple places, then I switched to a test id's.

This probably isn't perfect. But it should be much better and cause our tests to break less when we upgrade dependencies.

Reply:
As long as we have good reason to use the escape hatch, I'm happy using the escape hatch 👍🏻